test/client/fixtures/sandbox/event/focus-blur-change-test.js

```diff
@@ -261,6 +261,22 @@ asyncTest('without handlers', function () {
     testFocusing(0, startNext);
 });
 
+asyncTest('onfocus() must not be called on disabled input', function () {
```

Comment:
Rename to `asyncTest('disabled element', function () {`

Reply:
done